src/App.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
+
+ class App extends Component {
```

Comment:
please change this to a functional component, we will move away from class components